addons/packages/cartographer/0.3.0/package.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: cartographer.community.tanzu.vmware.com.0.3.0
```

Comment:
@cirocosta @kartiklunkad26 The name of the package and the imgpkgBundle.image field are different here than what is in the official 0.3.0 release of the package. Is this intentional?

I ran a `vendir sync` in the `addons/packages/cartographer` directory and got differences in the 0.3.0 package.yaml file.

The name in the PR is `cartographer.community.tanzu.vmware.com.0.3.0` while in the release it is `cartographer.tanzu.vmware.com.0.3.0`. The image also changes location from TCE's Harbor to Docker.

Can you confirm what should be here?

Reply:
@seemiller good catch! 

I just updated the PR with an updated asset structure for the releases we generate in `vmware-tanzu/package-for-cartographer` to better deal with handling multiple `cartographer-*` projects from there (described the problem here: https://github.com/vmware-tanzu/package-for-cartographer/issues/27)

with those changes (mentioned in the issue above) there'll be no further mismatches between `vendir sync`s here.

---

just to be sure: would it be possible to make use of `cartographer.tanzu.vmware.com` and images coming from dockerhub (actually, we'll be moving our public images to `ghcr.io` soon)? that'd certainly simplify some of the work there in `package-for-cartographer` 😅  specially the automation as (pls correct me if I'm wrong) we're not able to push to the registry from outside VMware infra.  (if we should stick with producing `cartographer.community.tanzu.vmware.com` and pushing to `projects.registry.tanzu...` that's fine, it'd just be easier the other way 😁 )

thank you!

Reply:
@cirocosta Names and OCI registries are up to your team's and organization's guidelines. I am ok with the name that you chose. And I believe that VMware would prefer our Harbor registry. However, our package policy does not specifically exclude others like GCR, GHCR or Docker Hub..